### PR TITLE
Homework: Nwd nnw

### PR DESCRIPTION
**Pull request by MalarzDawid**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @MalarzDawid

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,22 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+   int temp;
+
+   while (rhs!=0) {
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:6]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+       temp = rhs;
+       rhs = lhs%rhs;
+       lhs = temp;
+   }
+   if (lhs<0){
+       return lhs*(-1);
+   }
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:13]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
+   return lhs;
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if (lhs == 0 || rhs == 0) {
+        return 0;
+    }
+    return abs(lhs*rhs/NWD(lhs, rhs));
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:21]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**